sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -2242,11 +2250,13 @@
   /**
    * @since 5.2
    */
+  // TODO should be renamed to TECHNICALDEBT_RATIO_ON_NEW_CODE_KEY = "technicaldebt_ratio_on_new_code"
   public static final String NEW_SQALE_DEBT_RATIO_KEY = "new_sqale_debt_ratio";
 
   /**
    * @since 5.2
    */
+  // TODO should be renamed to TECHNICALDEBT_RATIO_ON_NEW_CODE
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1135)